File: frontend/src/app/shared/model/FlightResult.ts
import {Seat} from "./Seat";
import {Address} from "./Address";

export interface FlightResult {
  id: string,
  totalTicketPrize: number,
  seats: Seat[],
  departureAddress: Address,
  arrivalAddress: Address,
  departureTime: Date,
  departureDate: Date,
  arrivalTime: Date,
  arrivalDate: Date,
}

File: backend/JetSetGo.Application/Common/Model/FlightResult.cs
using JetSetGo.Domain.Flights.ValueObjects;

namespace JetSetGo.Application.Common.Model;

public class FlightResult
{
    public Guid Id { get; set; }
    public double  TotalTicketPrize { get; set; }
    public IEnumerable<SeatResult> Seats { get; set; } = null!;
    public Address DepartureAddress { get; set; } = null!;
    public Address ArrivalAddress { get; set; } = null!;
    public TimeOnly ArrivalTime { get; set; }
    public TimeOnly DepartureTime { get; set; }
    public DateOnly DepartureDate { get; set; }
    public DateOnly ArrivalDate { get; set; }
}